### PR TITLE
Update transform for submit

### PR DESCRIPTION
**Pull request by cehsu**
Transform should conform to:
```
{
  gi_bill_feedback: {
    form: // form as JSON string
  }
}
```

### CODE REVIEW

File: src/applications/edu-benefits/complaint-tool/helpers.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ function pollStatus(guid, onDone, onError) {
         if (!res || res.data.attributes.state === 'pending') {
           pollStatus(guid, onDone, onError);
         } else if (res.data.attributes.state === 'success') {
-          onDone(res.data.attributes.parsed_response);
+          onDone(res.data.attributes.parsedResponse);
```
Comment (cehsu) [inline src/applications/edu-benefits/complaint-tool/helpers.js]: @jholton will `X-Key-Inflection: camel` convert `parsed_response` to `parsedResponse`?

File: src/applications/edu-benefits/complaint-tool/helpers.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export function fetchInstitutions({ institutionQuery, page }) {
 export function transform(formConfig, form) {
   const formData = transformForSubmit(formConfig, form);
   return JSON.stringify({
-    educationBenefitsClaim: {
+    giBillFeedback: {
```
Comment (cehsu) [inline src/applications/edu-benefits/complaint-tool/helpers.js]: It looks like this is likely causing the 400 response, as the backend is expecting:
```
{
  gi_bill_feedback: {
    form: // form as JSON string
  }
}
```

File: src/applications/edu-benefits/complaint-tool/config/form.js [hunk 3]
```diff
@@ -272,96 +272,102 @@ const formConfig = {
           path: 'benefits-information',
           title: 'Benefits Information',
           uiSchema: {
-            programs: {
```
Comment (cehsu) [inline src/applications/edu-benefits/complaint-tool/config/form.js]: The schema structure was causing a 422. Nesting programs, assistance, and school within educationDetails resolves the error.

File: src/applications/edu-benefits/complaint-tool/helpers.js (hunk 1 above)
Comment (cehsu) [inline src/applications/edu-benefits/complaint-tool/helpers.js]: `X-Key Inflection: camel` formats this as camel case.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**